skxray/cdi.py

```diff
@@ -96,6 +98,27 @@ def gauss(dims, sigma):
     return y / np.sum(y)
 
 
+def _fft_helper(x, option='fft'):
```

Comment:
That is actually my original thought. However, the two helper functions are so similar, so I summarized them as one. If you strongly suggest that way, sure.